core/src/main/java/org/infinispan/remoting/transport/Transport.java

```diff
@@ -55,6 +55,38 @@
                                                                  boolean anycast) throws Exception;
 
    /**
+    * Asynchronously sends the {@link ReplicableCommand} to the destination using the specified {@link DeliverOrder}.
+    *
+    * @param destination  the destination's {@link Address}.
+    * @param rpcCommand   the {@link ReplicableCommand} to send.
+    * @param deliverOrder the {@link DeliverOrder} to use.
+    * @throws Exception if there was problem sending the request.
+    */
+   void sendTo(Address destination, ReplicableCommand rpcCommand, DeliverOrder deliverOrder) throws Exception;
+
+   /**
+    * Asynchronously sends the {@link ReplicableCommand} to the set of destination using the specified {@link
+    * DeliverOrder}.
+    *
+    * @param rpcCommand   the {@link ReplicableCommand} to send.
+    * @param deliverOrder the {@link DeliverOrder} to use.
+    * @param destinations the collection of destination's {@link Address}. If {@code null}, it sends to all the members
+    *                     in the cluster.
+    * @throws Exception if there was problem sending the request.
+    */
+   void sendTo(ReplicableCommand rpcCommand, DeliverOrder deliverOrder, Collection<Address> destinations) throws Exception;
+
+   /**
+    * Asynchronously sends the {@link ReplicableCommand} to all members in the cluster using the specified {@link
+    * DeliverOrder}.
+    *
+    * @param rpcCommand   the {@link ReplicableCommand} to send.
+    * @param deliverOrder the {@link DeliverOrder} to use.
+    * @throws Exception if there was problem sending the request.
+    */
+   void sendToAll(ReplicableCommand rpcCommand, DeliverOrder deliverOrder) throws Exception;
```

Comment:
Is this necessary? The other methods are fine with `destionations == null` meaning "all"